caos.ansible_roles/roles/docker_install/vars/main.yml

```diff
@@ -1 +1,4 @@
-docker_compose_version: 2.12.2
\ No newline at end of file
+docker_compose_version: 2.12.2
+suse_versions_wo_docker:
+  - "12.3"
+  - "12.4"
```

Comment:
What if we add support for a new distribution that also contains "12.3" in its name? Could we add the suse keyword in the check?

Reply:
These are only used in tasks that belong to SLES `install-docker-SLES.yaml` and  `install-docker-SLES_old.yaml` . It should not break the new distro ;-)